openquake/qa_tests_data/event_based_risk/case_miriam/job2.ini

```diff
@@ -5,7 +5,7 @@ calculation_mode = event_based
 
 [geometry]
 
-region_grid_spacing = 1
+region_grid_spacing = 1.25
 exposure_file = exposure_model.xml
```

Comment:
A scenario hazard job (or any other hazard job) with a `region` and `region_grid_spacing` might serve as a better test case for this functionality, rather than a risk job with an exposure and `region_grid_spacing`.

Reply:
True, but the case with also an exposure is the trickiest one and it deserves to be tested more. Also, most calculations are event based risk calculations, so I want to test this case more than any other.

Reply:
And you can always add a test in oq-risk-tests